base_exception/models/base_exception.py

```diff
@@ -191,9 +191,25 @@ class BaseException(models.AbstractModel):
         string='Main Exception',
         store=True,
     )
-    exception_ids = fields.Many2many('exception.rule', string='Exceptions')
+
+    exception_ids_formatted = fields.Text(
```

Comment:
I don't see this field being used anywhere... why?

Also I think the name is a bit confusing, as `_ids` suggests it's a x2many. You could change to:

```suggestion
    exceptions_description = fields.Text(
```

Reply:
It's not intended to be used in this module.
The purpose is to be used in the model that implements the exception. ie: https://github.com/OCA/sale-workflow/pull/878/files#diff-e3019ca8759cdf323e38ee35b80b147cR33